scipy/special/_add_newdocs.py

```diff
@@ -2754,20 +2754,22 @@ def add_newdoc(name, doc):
     .. math::
 
         G_n^{(p, q)}(x)
-          = \binom{2n + p - 1}{n}^{-1} P_n^{(p - q, q - 1)}(2x - 1),
+          = \frac{n!\Gamma(n+p)}{\Gamma(2n+p)} P_n^{(p - q, q - 1)}(2x - 1),
```

Comment:
Nitpick: assuming that `\hphanthom` works, we could align the terms in the fraction like this:

```suggestion
          = n! \frac{\Gamma(\hphantom{2}n+p)}{\Gamma(2n+p)} P_n^{(p - q, q - 1)}(2x - 1),
```

Reply:
While I can understand that in this way the difference between the arguments in the two gamma functions can be highlighted, I would consider this to be rather ugly from a typographical point of view. I would really prefer not to use `\hphantom` in this situation. Also, it makes sense to keep the factorial of n in the numerator, thereby reflecting the binomial structure of the prefactor. Unless you insist, I would thus prefer to keep the equation as it is.

Reply:
Dealer's choice :)